frontend/src/component/Account/PersonalData.jsx

```diff
@@ -73,26 +73,26 @@ export class PersonalData extends React.Component {
             />
           </div>
 
-          <div className='d-flex flex-wrap'>
-            <div className='d-inline-flex flex-column justify-content-start'>
+          <div>
+            <div>
               <input
-                className='personaldata__form__txtinput primaryColorBorderLighten form-control mt-3 mt-sm-0'
+                className='personaldata__form__txtinput__username primaryColorBorderLighten form-control mt-3 mt-sm-0'
```

Comment:
I know that the `mt-3 mt-sm-0` was not added in this PR, but maybe it's a good opportunity to add at `personaldata__form__txtinput__username` (check if this class it's not used in other context), don't you think?

Reply:
Done.